helix-core/src/main/java/org/apache/helix/PropertyPathBuilder.java

```diff
@@ -72,6 +74,7 @@
     typeToClassMapping.put(IDEALSTATES, IdealState.class);
     typeToClassMapping.put(CONFIGS, InstanceConfig.class);
     typeToClassMapping.put(EXTERNALVIEW, ExternalView.class);
+    typeToClassMapping.put(CUSTOMIZEDVIEW, CustomizedView.class);
```

Comment:
Is it overlapped with Meng's PR?
https://github.com/apache/helix/pull/851/files

Can we change this part as a pre-requisite change so both of you are on the same page.

Reply:
I looked at the other PR. They are exactly same. I think it makes sense to have both of them and one of us will resolve the conflict before the merge. It might take longer if we submit pre-requisite PR again. I will double check with Meng.